Packages/com.unity.inputsystem/InputSystem/Actions/InputActionAsset.cs

```diff
@@ -946,10 +947,12 @@ private void OnDestroy()
         [NonSerialized] internal InputActionRebindingExtensions.ParameterOverride[] m_ParameterOverrides;
 
         [NonSerialized] internal InputActionMap.DeviceArray m_Devices;
+        [SerializeField] internal int m_Version;
```

Comment:
The introduced "version" seems related to the the file format and not the asset object right? Hence I would recommend removing this field. I.e. in a code-only project version doesn't play any role, hence its misplaced IMO and we would better not associate it with the asset.